src/Backend/Controllers/SiteController.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace App\Backend\Controllers;
+namespace App\Backend\Controller;
```

Comment:
Directory name should be renamed as well.

Reply:
:) magic github buttons...
will be done